docs/language/crypto.mdx

```diff
@@ -102,6 +102,12 @@ struct PublicKey {
         domainSeparationTag: String,
         hashAlgorithm: HashAlgorithm
     ): Bool
+
+    /// Verifies the proof of possession of the private key.
+    /// This function is only implemented if the signature algorithm of the public key is BLS, it errors if called with any other signature algorithm. 
+    pub fun verifyPoP(
+        proof: [UInt8],
+    ): Bool
```

Comment:
`PublicKeyVerifyPoPFunctionType` has `Label` `ArgumentLabelNotRequired`, also trailing commas are not supported:

```suggestion
    pub fun verifyPoP(_ proof: [UInt8]): Bool
```